### ISSUE TITLE
Including DataPortalController in Blazor.Server

### ISSUE THREAD

**Issue by animjn** [MarimerLLC/cslaforum#936]
Do you foresee any issues with including DataPortalController in Blazor Server Project - is there a  possibility of conflict in ApplicationContext.User if I combine ProjectTracker.AppServerCore and ProjectTracker.Ui.Blazor.Server in an attempt to reduce the number of .NET Core apps?  

**Version and Platform**
CSLA version: 5.1
Platform: Blazor Server

**Comment by rockfordlhotka** [comment]
No, there should be no problem hosting a data portal endpoint in an ASP.NET Core site, regardless of whether it is also hosting server-side Blazor or not.

**Comment by animjn** [comment]
Hi! To achieve this (consolidate the two), when I add _services.AddHttpContextAccessor()_, which is otherwise not present in _Startup.cs_ of _ProjectTracker.AppServerCore_, and try to access Dataportal I receive _DataPortalException... at Csla.DataPortalClient.HttpProxy.Update_

If I add _Csla.ApplicationContext.WebContextManager = null_ in _DataPortalController_ it would continue to work normally.

I'd like to get your thoughts on this. Could I be missing something?

**Comment by rockfordlhotka** [comment]
I think this might be fixed in 5.2.

At least the ProjectTracker I have in master has no problem when I add `services.AddHttpContextAccessor()` in the configuration.

Though I don't recall this problem existing in 5.1 either, but version 5.2 is when client-side Blazor will be fully supported.

**Comment by rockfordlhotka** [comment]
Ahh, I _am_ able to repro the issue in the `BlazorExample` sample - so there's definitely something going on.

**Comment by rockfordlhotka** [comment]
I figured it out (I think). The problem is a CORS issue - you need the CORS configuration to allow the Blazor client to call the server.

This must be a change in the latest Blazor preview, because the `BlazorExample` app _did work_, but now does not - regardless of whether the `AddHttpContextAccessor` call is made or not.
